src/main/java/ch/heigvd/pro/b04/auth/LoginController.java

```diff
@@ -28,14 +32,16 @@ public LoginController(ModeratorRepository moderators) {
    */
   @RequestMapping(value = "auth", method = RequestMethod.POST)
   @ResponseBody
-  public TokenCredentials login(@RequestBody UserCredentials credentials)
+  public String login(@RequestBody UserCredentials credentials)
```

Comment:
The endpoint return type should not be changed.

Reply:
But we have to enhance our product, it may require to modify that kind of things

Reply:
Why ? What enhancement was added ?